uwblueprint/project-read-frontend
# Login with Firebase auth

### Notion ticket link



[Firebase sign in](https://www.notion.so/uwblueprintexecs/Firebase-sign-in-8c80a086541145718caefb6553810f3d)



### Implementation description

- Connect with Firebase and redirects to Firebase stub login page on `/login`
- User is stored in `AuthContext`



### Steps to test

1. Create a user in Firebase
2. go to `/login` and log in with the user's credentials. It should show the email on the registration stub page



### What should reviewers focus on?

- The auth provider
